entrypoint.sh

```diff
@@ -36,8 +36,8 @@ done
 # asure that results folder is readable
 chmod -R 755 results
 
-# Schedule for TEST_SUITE=big
-echo "03 7,9,11,14,17,18,19,22,0 * * * TEST_SUITE=big /app/run_tests.sh" >> /etc/crontabs/root
+# Schedule for TEST_SUITE=big every two hours
+echo "*/2 * * * TEST_SUITE=big /app/run_tests.sh" >> /etc/crontabs/root
 
 # Schedule for TEST_SUITE=small
```

Comment:
maybe we could also explicity write the frequency on this comment for consistency?
```suggestion
# Schedule for TEST_SUITE=small every 5 minutes
```